_layouts/slack.html

```diff
@@ -24,61 +24,18 @@
             <h3>Already a Team member?</h3>
             <p>click here: <a href="https://openwhisk-team.slack.com/"><b>openwhisk-team.slack.com</b></a></p>
             <h3>If not, join us!</h3>
-            <p>Join the OpenWhisk Community to discuss the project in real-time.</p>
+            <p>Join the OpenWhisk Community to discuss the project in real-time.
+            </p>
```

Comment:
This might not be intended.

Reply:
fixed.  I was going to add something here, but ended up doing it below.